src/components/trends/smbg/SMBGAvg.js

```diff
@@ -0,0 +1,78 @@
+/*
+ * == BSD2 LICENSE ==
+ * Copyright (c) 2016, Tidepool Project
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the associated License, which is identical to the BSD 2-Clause
+ * License as published by the Open Source Initiative at opensource.org.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the License for more details.
+ *
+ * You should have received a copy of the License along with this program; if
+ * not, you can obtain one from Tidepool Project at tidepool.org.
+ * == BSD2 LICENSE ==
+ */
+
+import React, { PropTypes } from 'react';
+
+import styles from './SMBGAvg.css';
+
+const SMBGAvg = (props) => {
+  const { datum } = props;
+  if (!datum) {
+    return null;
+  }
+
+  const { focusAvg, avgRadius, unfocusAvg: unfocus, xScale, yPositions } = props;
+  const xPos = xScale(datum.msX);
+  const focus = () => {
+    focusAvg(datum, {
+      left: xPos,
+      tooltipLeft: datum.msX > props.tooltipLeftThreshold,
+      yPositions,
+    });
+  };
+
+  return (
+    <g id={`smbgAvgGroup-${datum.id}`}>
```

Comment:
I only used a <g> in the combined version of range and average because you have to return only a single element in JSX. Now that there's only a single child, the <g> is unnecessary, so it'd be best to remove it.